apps/arm/include/ClawConfig.h

```diff
@@ -36,13 +36,14 @@ static Controller::Velocity vel(motor, encoder, std::nullopt, velPID, POLOLUMADE
                                 LIM_CLAW_OPEN, NC);
 static Controller::Position pos(motor, encoder, std::nullopt, posPID, POLOLUMADEGPERSEC, POLOLUMAXCURRENT,
                                 LIM_CLAW_OPEN, NC);
-static Controller::Current cur(motor, encoder, std::nullopt, curPID, POLOLUMADEGPERSEC, POLOLUMAXCURRENT, LIM_CLAW_OPEN,
-                               NC);
+// static Controller::Current cur(motor, encoder, std::nullopt, curPID, POLOLUMADEGPERSEC, POLOLUMAXCURRENT,
```

Comment:
i had to comment out the current controllers, bc we dont have a current sensor driver yet, and current controller no longer takes in optional current sensor